package.json

```diff
@@ -3,20 +3,31 @@
   "version": "1.0.1",
   "description": "@astrojs/compiler to process synchronously.",
   "main": "lib/index.cjs",
-  "types": "lib/index.d.ts",
+  "types": "lib/index.d.cts",
   "files": [
     "lib",
     "browser"
   ],
   "exports": {
     ".": {
-      "browser": "./browser/index.mjs",
-      "import": "./lib/index.mjs",
-      "require": "./lib/index.cjs",
-      "types": "./lib/index.d.ts"
+      "browser": {
+        "types": "./browser/index.d.ts",
+        "default": "./browser/index.js"
+      },
+      "import": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.js"
+      },
+      "require": {
+        "types": "./lib/index.d.ts",
+        "default": "./lib/index.cjs"
+      }
     },
-    "./package.json": "./package.json",
-    "./browser/index.mjs": "./browser/index.mjs"
+    "./browser": {
```

Comment:
1. We've alreay been using `type: module`, so `.mjs` is not required
2. `"./browser/index.js"` auto infers `"./browser/index.d.ts"` as its type, which is better?
3. Using `./browser/index.mjs` entry seems too long for end users, so I changed to simple `./browser`